pages/index.tsx

```diff
@@ -1,75 +1,347 @@
+              <Image
+                src="/screenshots/screen_main.jpg"
+                width="1500"
+                height="948"
+                layout="responsive"
+              />
+            </div>
+          </div>
         </div>
-
-        <div className="mt-5 sm:mt-8 sm:flex sm:justify-center lg:justify-start">
-          <div className="rounded-md shadow">
-            <a
-              href="https://docs.overseerr.dev/getting-started/installation"
-              className="flex items-center justify-center w-full px-6 py-3 text-base font-medium text-white transition bg-indigo-600 border border-transparent rounded-md hover:bg-indigo-500 md:py-4 md:text-lg md:px-8"
-            >
-              <svg
-                className="w-6 h-6 mr-2"
-                fill="none"
-                stroke="currentColor"
-                viewBox="0 0 24 24"
-                xmlns="http://www.w3.org/2000/svg"
-              >
-                <path
-                  strokeLinecap="round"
-                  strokeLinejoin="round"
-                  strokeWidth={2}
-                  d="M4 16v1a3 3 0 003 3h10a3 3 0 003-3v-1m-4-4l-4 4m0 0l-4-4m4 4V4"
+      </div>
+      <div className="relative pb-32 overflow-hidden md:pt-16">
+        <div className="relative">
+          <div className="lg:mx-auto lg:max-w-7xl lg:px-8 lg:grid lg:grid-cols-2 lg:grid-flow-col-dense lg:gap-24">
+            <div className="max-w-xl px-4 mx-auto sm:px-6 lg:py-16 lg:max-w-none lg:mx-0 lg:px-0">
+              <div>
+                <div>
+                  <span className="flex items-center justify-center w-12 h-12 bg-indigo-600 rounded-md">
+                    <SparklesIcon
+                      className="w-6 h-6 text-white"
+                      aria-hidden="true"
+                    />
+                  </span>
+                </div>
+                <div className="mt-6">
+                  <h2 className="text-3xl font-extrabold tracking-tight text-gray-200">
+                    The best way to discover media
+                  </h2>
+                  <p className="mt-4 text-lg text-gray-400">
+                    Overseerr helps you find media you{' '}
+                    <em className="font-bold">want</em> to watch. With inline
+                    recommendations and suggestions, you will find yourself
+                    deeper and deeper in a rabbit hole of content you never knew
+                    you just had to have.
+                  </p>
+                </div>
+              </div>
+              <div className="pt-6 mt-8 border-t border-gray-700">
+                <blockquote>
+                  <div>
+                    <p className="text-base text-gray-400">
+                      Media pages display organized, easy-to-digest information.
+                      We show you the information you care about seeing.
+                      RottenTomatoes scores, the cast and crew, and more.
```

Comment:
```suggestion
                      Media pages display organized, easy-to-digest information.
                      We show you the information you care about seeing:
                      RottenTomatoes scores, the cast and crew, and more.
```
An ellipsis would also work here.